05_book_titles/book.rb

```diff
@@ -1,3 +1,28 @@
 class Book
-# write your code here
+	attr_accessor :title
+
+	public
+	def title=(name)
```

Comment:
this method (line 5-7) doesn't make sense here. if you use `attr_accessor` you don't need another setter here, but you do need an initializer:
```
def initialize(title)
  @title = title
end
```